shared/featureconfig/flags.go

```diff
@@ -0,0 +1,19 @@
+package featureconfig
+
+import (
+	"github.com/urfave/cli"
+)
+
+var (
+	// VerifyAttestationSigsFlag determines whether to verify signatures for attestations.
+	VerifyAttestationSigsFlag = cli.BoolFlag{
+		Name:  "enable-attestation-signature-verification",
+		Usage: "Verify signatures for attestations (testing)",
```

Comment:
why `(testing)` at the end?

Reply:
yeah I don't think you need to include a suffix like this, feature flags are assumed to be experimental